app/src/main/res/raw/segment_postprocess.glsl

```diff
@@ -0,0 +1,13 @@
+#extension GL_OES_EGL_image_external : require
```

Comment:
Rename this file to segment_postprocess_shader.glsl

Reply:
done